### PR TITLE
Update persistent-volumes.md

### PR DESCRIPTION
**Pull request by fearlesschenc**
change use by to used by

### CODE REVIEW

**Comment by makoscafee** [comment]
Hi @fearlesschenc thank you the changes look good, mind signing `CLA` so your changes can be merged?.

**Comment by RA489** [comment]
/check-cla

Rest LGTM

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Pods use claims as volumes. The cluster inspects the claim to find the bound vol
 Once a user has a claim and that claim is bound, the bound PV belongs to the user for as long as they need it. Users schedule Pods and access their claimed PVs by including a `persistentVolumeClaim` in their Pod's volumes block. [See below for syntax details](#claims-as-volumes).
 
 ### Storage Object in Use Protection
-The purpose of the Storage Object in Use Protection feature is to ensure that Persistent Volume Claims (PVCs) in active use by a Pod and Persistent Volume (PVs) that are bound to PVCs are not removed from the system, as this may result in data loss.
+The purpose of the Storage Object in Use Protection feature is to ensure that Persistent Volume Claims (PVCs) in active used by a Pod and Persistent Volume (PVs) that are bound to PVCs are not removed from the system, as this may result in data loss.
```
Comment (msau42) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Sorry I think grammar-wise, "use" is the correct form (it's used as a noun in this context)

**Comment by RA489** [comment]
@fearlesschenc do you have any problem while signing the CLA?

**Comment by jimangel** [comment]
@fearlesschenc I'm going to close this PR for now, feel free to open once you get the email situation / CLA resolved.

Let me know if you have any questions!

/close

**Pull request closed without merging**